src/Molecules/ContestComponent/ContestComponentStyled.js

```diff
@@ -9,10 +10,12 @@ const ContestComponentStyled = styled.div`
   flex-direction: column;
   justify-content: space-around;
   padding: 1.25rem 1.25rem 1.375rem 1.25rem;
-  border: solid 1px ${COLORS.grey200};
+  border: solid 1px
+    ${(props) =>
+      props.colorMotive === c.GREEN ? COLORS.positive600 : COLORS.grey200};
```

Comment:
You should change border,as in figma 
![image](https://user-images.githubusercontent.com/56206231/151601995-bf4cd5ec-6d4e-4028-8467-21636691a59d.png)